README.md

```diff
@@ -1760,6 +1764,8 @@ zinit as"null" wait"1" lucid for \
             zdharma-continuum/zsh-diff-so-fancy \
     sbin"git-url;git-guclone" make"GITURL_NO_CGITURL=1" \
             zdharma-continuum/git-url
+# NOTICE: zdharma/zsh-diff-so-fancy is still not available
```

Comment:
```suggestion
```

Reply:
It won't be available in the future either. I don't want to distribute it -- it doesn't make sense to me

Reply:
maybe we need remove this plugin in code in same time

Reply:
I didn't port it over. That is "removing it."

Reply:
oh, I see what you meant